src/sql/postgres/DataCell.zig

```diff
@@ -927,7 +927,7 @@ pub const Putter = struct {
             // The indexed columns can be out of order.
             .index => |i| i,
 
-            else => @intCast(index),
+            else => @truncate(index),
```

Comment:
index is already u32 no need to intCast or truncate
```suggestion
            else => index,
```